pkg/repo/chartrepo.go

```diff
@@ -271,3 +274,8 @@ func ResolveReferenceURL(baseURL, refURL string) (string, error) {
 	parsedBaseURL.Path = strings.TrimSuffix(parsedBaseURL.Path, "/") + "/"
 	return parsedBaseURL.ResolveReference(parsedRefURL).String(), nil
 }
+
+func (e *Entry) String() string {
+	buf, _ := json.Marshal(e)
```

Comment:
We should probably `panic` here if an error occurs while marshaling. It's very unlikely to occur, but we shouldn't silently ignore the error either.

Reply:
You are right. In case it fails there it should exit immediately. Fixed that in the latest commit!